### DISCUSSION TITLE
Can I use my ComfyUI running under Windows and A8R8 under WSL2?

### DISCUSSION THREAD

**Discussion by NeoAnthropocene** [ramyma/a8r8#21] [Q&A] [answered]
I have whole setup (A1111/Froge/ComfyUI) running under windows 11.

If I setup the WSL version of A8R8, can I use it with the ComfyUI running under Win11?

**Comment by ramyma** [comment, marked as answer]
Yes, running any of those backends on the local Windows (10/11) setup is the intended use case; the installer sets up the address for you.

If you encounter access issues, it might require adding a firewall rule to allow access to Windows network from within WSL.

You can also connect to a remote server by changing the address for the required backend in `user.sh`

Does this answer your question?
